### PR TITLE
Brltty

### PR DESCRIPTION
**Pull request by bramd**
Fixes the issues as described in nixos/nixpkgs#114745.

### CODE REVIEW

File: nixos/modules/services/hardware/brltty.nix [hunk 1]
```diff
@@ -10,14 +10,6 @@ let
     "rescue.target" "emergency.target"
   ];
 
-  genApiKey = pkgs.writers.writeDash "generate-brlapi-key" ''
```
Comment (rnhmjoj) [inline nixos/modules/services/hardware/brltty.nix]: Is this intentional? You may have to rebase your PR...

**Comment by rnhmjoj** [comment]
Thank you!

**Comment by bramd** [comment]
I just found and fixed another problem where the API bindings, such as 
those for Python, were looking at the wrong location for an API keyfile.

**Comment by rnhmjoj** [comment]
@bramd if you have more fixes, please open another PR or post a patch and I'll include it.

**Pull request merged**